.github/workflows/build.yaml

```diff
@@ -23,8 +23,8 @@ jobs:
 
       - uses: Open-CMSIS-Pack/gen-pack-action@main
         with:
-          doxygen-version: 1.9.6                                  # default
-          packchk-version: 1.3.95                                 # default
+          doxygen-version: 1.13.2                                 # default
+          packchk-version: 1.4.2                                  # default
           gen-doc-script: ./DoxyGen/gen_doc.sh                    # skipped by default
           check-links-script: ./DoxyGen/check_links.sh            # skipped by default
```

Comment:
check-links-script is deprecated in gen-pack lib 0.11.3
Instead, we call check_links.sh script as part of gen_doc.sh.
For example:
https://github.com/ARM-software/CMSIS_6/blob/eb5eeaacad826fbbe30c54fe67e66df7dcecbfa8/CMSIS/Documentation/Doxygen/gen_doc.sh#L70
https://github.com/ARM-software/CMSIS_6/blob/eb5eeaacad826fbbe30c54fe67e66df7dcecbfa8/CMSIS/Documentation/Doxygen/gen_doc.sh#L138